modules/core/src/passes/pick-layers-pass.js

```diff
@@ -57,7 +56,8 @@ export default class PickLayersPass extends LayersPass {
         blend: !pickZ
       },
       () => {
-        this.drawLayers({
+        super.render({
```

Comment:
Nit: Any chance we can avoid introducing a call to `super`? What happens if we call `this.render()`? If we rename `super.render()`?